src/gc/gcenv.ee.standalone.inl

```diff
@@ -11,166 +11,196 @@
 // will be fowarded to this interface instance.
 extern IGCToCLR* g_theGCToCLR;
 
+// A note about this:
+// In general, we don't want to pretend to be smarter than the compiler
+// and force it to inline things. However, inlining is here is required
+// for correctness as it stands today (though it will not always be required).
```

Comment:
When it is not going to be required?

This does not make sense ... there are two build configurations:

- Statically linked GC: This file should not be compiled in for statically linked GC at all.
- Standalone GC: The GCToEEInterface* symbols should be private in standalone GC .so, so I do not see why is the linked mixing them with the ones from VM.

What am I missing?